docs/admin/troubleshooting/system-tables.rst

```diff
@@ -1,14 +1,15 @@
 .. highlight:: psql
 .. _systables:
 
-===============================
-Troubleshooting with sys-tables
-===============================
+==============================
+Diagnostics with System Tables
+==============================
 
 CrateDB maintains a set of diagnostic tables in the **sys** schema. It
 currently consists of ten tables that provide an overview of the cluster state.
```

Comment:
> It currently consists of ten tables that provide an overview of the cluster state.

Not newly added in this PR, but while being at it, I suggest removing this reference to the number of tables. There are 20 tables in `sys` right now, but I don't think the exact number is of any value to the reader.

Reply:
Suggestion?

> CrateDB maintains a set of diagnostic tables in the **sys** schema, which provide insights into the cluster state.

Reply:
Sounds good!

Reply:
Updated with ba450ae634.